### ISSUE TITLE
image caption not shown if followed by footnote

### ISSUE THREAD

**Issue by colcord** [jgm/pandoc-citeproc#187]
```
![caption with citation as footnote after image link, web image](http://imgs.xkcd.com/comics/wikipedian_protester.png)^[@_xkcd citation] 
```

Result: image in pdf followed by footnote number, but no caption.

Expected: image, with caption below, caption followed by footnote number, footnote contains citation

(This is one bug in a test suite trying to find a way to have citations in image captions with a non default citation style.)

bib file:

@online{_xkcd,
  title = {xkcd: {Wikipedian} {Protester}},
  url = {https://xkcd.com/285/},
  timestamp = {2015-10-30T07:53:50Z},
  urldate = {2015-10-30},
  note = {00000},
  file = {xkcd\: Wikipedian Protester:/home/frank/Zotero/storage/SM3Q5I5G/285.html:}
}

**Comment by jgm** [comment]
This isn't a bug.  Pandoc's documentation says that a figure is created when the image is the only thing in a paragraph.  When you add a footnote, you have an image + a footnote, so you get a regular paragraph.

**Comment by colcord** [comment]
Sorry to hear you've closed this.

Is there is no work around for that definition of a paragraph?
I was hoping to offer this as an alternative if the citation in a footnote
style can't be inside captions.

Or, do you run into a problem in the list of figures after the table of
contents, if the figure captions have footnotes following? Could you split
the caption and footnote in this context?

thanks again

On 1 November 2015 at 21:46, John MacFarlane <notifications@github.com>
wrote:

> Closed #187 <https://github.com/jgm/pandoc-citeproc/issues/187>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jgm/pandoc-citeproc/issues/187#event-451454422>.
>



-- 
frank@colcord.net, cell: +447763116793, skype: colcord,
if you ever get no response, please send to: fcolcord@gmail.com

**Comment by jgm** [comment]
It doesn't make sense to attach a footnote to a whole figure
environment anyway.  The way forward would be to find
some way to make footnotes in a caption work
(jgm/pandoc#1506).

**Comment by jgm** [comment]
Note that jgm/pandoc#1506 is now completely fixed, and your
original test case with modern-humanities-association.csl
now works with footnotes in the caption.

+++ colcord [Nov 01 15 14:32 ]:

**Comment by colcord** [comment]
Thank you so much. I can't wait to see it working.

On 1 November 2015 at 23:31, John MacFarlane <notifications@github.com>
wrote:

> Note that jgm/pandoc#1506 is now completely fixed, and your
> original test case with modern-humanities-association.csl
> now works with footnotes in the caption.
>
> +++ colcord [Nov 01 15 14:32 ]:
> > Sorry to hear you've closed this.
> > Is there is no work around for that definition of a paragraph?
> > I was hoping to offer this as an alternative if the citation in a
> > footnote
> > style can't be inside captions.
> > Or, do you run into a problem in the list of figures after the table of
> > contents, if the figure captions have footnotes following? Could you
> > split
> > the caption and footnote in this context?
> > thanks again
> > On 1 November 2015 at 21:46, John MacFarlane <notifications@github.com>
> > wrote:
> > > Closed #187 <https://github.com/jgm/pandoc-citeproc/issues/187>.
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <https://github.com/jgm/pandoc-citeproc/issues/187#event-451454422>.
> > >
> > --
> > frank@colcord.net, cell: +447763116793, skype: colcord,
> > if you ever get no response, please send to: fcolcord@gmail.com
> >
> > —
> > Reply to this email directly or [1]view it on GitHub.
> >
> >References
> >
> > 1.
> https://github.com/jgm/pandoc-citeproc/issues/187#issuecomment-152870557
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jgm/pandoc-citeproc/issues/187#issuecomment-152876519>
> .
>



-- 
frank@colcord.net, cell: +447763116793, skype: colcord,
if you ever get no response, please send to: fcolcord@gmail.com
